plugins/clipboard/plugin.js

```diff
@@ -1685,6 +1740,30 @@
 	 * @class CKEDITOR.plugins.clipboard
 	 */
 	CKEDITOR.plugins.clipboard = {
+		/**
+		 * Adds a file matcher verifying if a file should be supported via clipboard operations.
+		 *
+		 * In the case of pasting or dragging and dropping unsupported files,
+		 * the clipboard plugin will show a notification informing a user that a given file type is not supported.
+		 *
+		 * ```javascript
+		 * CKEDITOR.plugins.clipboard.addFileMatcher( editor, function( file ) {
+		 * 		var supportedImageTypes = [ 'image/png', 'image/jpeg', 'image/gif' ];
+		 * 		return CKEDITOR.tools.indexOf( supportedImageTypes, file.type ) !== -1;
+		 * } );
```

Comment:
Shouldn't it be indented only once? :thinking: 

```suggestion
		 * CKEDITOR.plugins.clipboard.addFileMatcher( editor, function( file ) {
		 * 	var supportedImageTypes = [ 'image/png', 'image/jpeg', 'image/gif' ];
		 * 	return CKEDITOR.tools.indexOf( supportedImageTypes, file.type ) !== -1;
		 * } );
```